easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -465,29 +472,28 @@ def run_pip_check(log, python_cmd, unversioned_packages):
     for unversioned_package in unversioned_packages:
         try:
             faulty_pkg_names.remove(unversioned_package)
-            log.debug('Excluding unversioned package %s from check', unversioned_package)
+            log.debug(f"Excluding unversioned package '{unversioned_package}' from check")
         except ValueError:
             try:
                 version = next(pkg['version'] for pkg in pkgs if pkg['name'] == unversioned_package)
             except StopIteration:
-                msg = ('Package %s in unversioned_packages was not found in the installed packages. '
-                       'Check that the name from `python -m pip list` is used which may be different '
-                       'than the module name.' % unversioned_package)
+                msg = f"Package '{unversioned_package}' in unversioned_packages was not found in "
```

Comment:
@boegel What was wrong with the original syntax? I find the repetition of the variable name overly verbose and IMO introduces a new source for errors. It seems like PEP even recommends this, although I have only found a [secondary source](https://docs.astral.sh/ruff/rules/multi-line-implicit-string-concatenation/)
> [PEP 8](https://peps.python.org/pep-0008/#maximum-line-length) recommends the use of implicit string concatenation within parentheses